receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -219,35 +231,147 @@ func TestTranslateV2(t *testing.T) {
-				rmAttributes1.PutStr("service.instance.id", "107cn001")
+				parseJobAndInstance(rm1.Resource().Attributes(), "service-x/test", "107cn001")
 				sm1 := rm1.ScopeMetrics().AppendEmpty()
+				sm1.Scope().SetName(defaultBuildName)
+				sm1.Scope().SetVersion(defaultBuildVersion)
+				// Expect 2 separate gauge metrics, one per timeseries.
 				sm1Attributes := sm1.Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty().Attributes()
 				sm1Attributes.PutStr("d", "e")
 				sm1Attributes.PutStr("foo", "bar")
-				// Since we don't check "scope_name" and "scope_version", we end up with duplicated scope metrics for repeated series.
-				// TODO: Properly handle scope metrics.
 				sm2Attributes := sm1.Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty().Attributes()
 				sm2Attributes.PutStr("d", "e")
 				sm2Attributes.PutStr("foo", "bar")
 
+				// For job "foo" with instance "bar", fallback to BuildInfo.
 				rm2 := expected.ResourceMetrics().AppendEmpty()
-				rmAttributes2 := rm2.Resource().Attributes()
-				rmAttributes2.PutStr("service.name", "foo")
-				rmAttributes2.PutStr("service.instance.id", "bar")
-				mAttributes2 := rm2.ScopeMetrics().AppendEmpty().Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty().Attributes()
-				mAttributes2.PutStr("d", "e")
-				mAttributes2.PutStr("foo", "bar")
-
+				parseJobAndInstance(rm2.Resource().Attributes(), "foo", "bar")
+				sm2 := rm2.ScopeMetrics().AppendEmpty()
+				sm2.Scope().SetName(defaultBuildName)
+				sm2.Scope().SetVersion(defaultBuildVersion)
+				m3 := sm2.Metrics().AppendEmpty().SetEmptyGauge()
+				dp3 := m3.DataPoints().AppendEmpty()
+				dp3.Attributes().PutStr("d", "e")
+				dp3.Attributes().PutStr("foo", "bar")
+				return expected
+			}(),
+			expectedStats: remote.WriteResponseStats{},
+		},
+		{
+			name: "provided otel_scope_name and otel_scope_version",
+			request: &writev2.Request{
+				Symbols: []string{
+					"", "__name__", "metric_with_scope",
+					"otel_scope_name", "custom_scope",
+					"otel_scope_version", "v1.0",
+					"job", "service-y/custom",
+					"instance", "instance-1",
+				},
+				Timeseries: []writev2.TimeSeries{
+					{
+						Metadata:   writev2.Metadata{Type: writev2.Metadata_METRIC_TYPE_GAUGE},
+						LabelsRefs: []uint32{1, 2, 3, 4, 5, 6, 7, 8, 9, 10},
+						Samples:    []writev2.Sample{{Value: 10, Timestamp: 100}},
+					},
+				},
+			},
+			expectedMetrics: func() pmetric.Metrics {
+				expected := pmetric.NewMetrics()
+				rm := expected.ResourceMetrics().AppendEmpty()
+				parseJobAndInstance(rm.Resource().Attributes(), "service-y/custom", "instance-1")
+				sm := rm.ScopeMetrics().AppendEmpty()
+				sm.Scope().SetName("custom_scope")
+				sm.Scope().SetVersion("v1.0")
+				_ = sm.Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty()
+				return expected
+			}(),
+			expectedStats: remote.WriteResponseStats{},
+		},
+		{
+			name: "missing otel_scope_name/version falls back to BuildInfo",
```

Comment:
We don't need a fallback :) 

But we could certify that all metrics with a missing scope would end up within the same scope with no attributes.